chore: update http dashboard panels

lzap opened the pull request:
I took a closer look on our SLO target (200ms successful requests) and because it way way bellow 95% I investigated, looks like the query we have there is not correct. Also took the opportunity to slightly enhance all other HTTP-related panels:

* Removed HTTP Success Latency graph and created a new one named HTTP Success Latency 200ms Target which is now the "big number" with background line graph for history.
* The expression `sum(rate(provisioning_http_request_duration_ms_bucket{code=~"2.*", le="200"}[$__range])) / sum(rate(provisioning_http_request_duration_ms_count[$__range]))` does divide by all requests, but we need to divide only by successful counts. This was giving us quite low number under our current SLO (95%). The correct expression is `sum(rate(provisioning_http_request_duration_ms_bucket{code=~"2.*", le="200"}[$__range])) / sum(rate(provisioning_http_request_duration_ms_bucket{code=~"2.*", le="+Inf"}[$__range]))` which divides 200ms successful requests by all successful requests. Either infinity bucket or `_count` can be used, this is equal.
* Updated HTTP requests color to blue (it is informative only).
* Removed decimals from HTTP Success Rate, added background line graph for history.
* Renamed legends in HTTP Throughtput Rate, updated y-axis to ops/m and made line 2px width for better visibility of the red line.
* Changed the Request Latency to point graph (it was already sort of point with lines).
* Renamed Source Avail Success Rate to Source Avail Delivery rate, that represents the number in a more correct way.
* Rewritten Source Availability Check Latency to be dot graph similarly to the HTTP Request Latency.

Before the patch:

![image](https://user-images.githubusercontent.com/49752/235897763-88c48b3c-f93f-41ee-a38a-76d89d2c52a2.png)

After the patch:

![image](https://user-images.githubusercontent.com/49752/235897682-d8d41525-7886-4190-8e99-a9f5b65956fb.png)

lzap commented on deploy/Provisioning-1674548289785.json:
```diff
@@ -1553,8 +1654,8 @@
-          "text": "crcs02ue1-prometheus",
-          "value": "crcs02ue1-prometheus"
+          "text": "crcp01ue1-prometheus",
+          "value": "crcp01ue1-prometheus"
```
Ha this was not intentional, I made the changes on prod Grafana this needs to be fixed.

lzap commented on deploy/Provisioning-1674548289785.json:
```diff
@@ -24,6 +24,7 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
+  "id": 66892,
```
Not sure why "id" is being added here.

ezr-ondrej replied:
did you figure out?

lzap commented on deploy/Provisioning-1674548289785.json:
```diff
@@ -746,7 +747,7 @@
                 "id": "color",
                 "value": {
-                  "fixedColor": "red",
+                  "fixedColor": "orange",
```
I don’t like red color for p50 dots, red means "bad" so I am proposing orange here. No other colors from the default color palettes are available.

ezr-ondrej reviewed (approved):
I'm guessing you need to regenerate the configmap, apart of that looks really great! thanks for doing this! :orange_heart:

Nit: did you find out why the id is added?
Nit2: why is the 99% in first pane lower line than 96% in the second pane? :thinking:

lzap commented:
> id

No idea why this was added. I removed it just to be sure.

> sparkline problem

No idea how to set y-axis on a sparkline, I do have "max" value set to 1 for both, yet, something is a bit off. I just noticed tho that one panel was slightly wider, changed, it is now the same. Also added sparkline to the SAC Delivery rate:

![image](https://user-images.githubusercontent.com/49752/235920115-a26f956e-3957-4a32-b97f-fd9ac6a0e079.png)

lzap commented:
Regenerated, rebased.

ezr-ondrej commented:
Thanks! :orange_heart:
